### PR TITLE
dark/light theme switch, metamask connection & bugfix typography comp…

### PR DESCRIPTION
**Pull request by 0xnikter**

### CODE REVIEW

File: app/src/App.css [hunk 1]
```diff
@@ -15,14 +18,15 @@
 
 .App-header {
   /* background-color: #282c34; */
-  background: linear-gradient(264.99deg,#9f329f -52.92%,#1161fe 95.58%);
+  /*background: linear-gradient(264.99deg,#9f329f -52.92%,#1161fe 95.58%);*/
```
Comment (MathisGD) [inline app/src/App.css]: You can delete this, as we don't need it anymore.

File: app/src/App.css [hunk 2]
```diff
@@ -15,14 +18,15 @@
...
+  background: var(--background);
   min-height: 100vh;
   display: flex;
   flex-direction: column;
   /* align-items: center; */
   /* justify-content: space-between; */
   font-size: calc(10px + 2vmin);
-  color: white;
+  /*color: white;*/
```
Comment (MathisGD) [inline app/src/App.css]: Same

File: app/src/App.js [hunk 3]
```diff
@@ -1,16 +1,23 @@
...
-      <header className="App-header">
-        <Navbar />
-        <div className="Content">
-          <Bridge />
-        </div>
-      </header>
+        <MetaMaskProvider>
+            <header className="App-header">
+                <Navbar />
+                <ThemeSwitch/>
```
Comment (MathisGD) [inline app/src/App.js]: I don't know what you think, but I would prefer it to be in the navbar (`ThemeSwitch`)

File: app/src/components/navbar.module.css [hunk 4]
```diff
@@ -1,6 +1,7 @@
 .navbar {
-    background: linear-gradient(264.99deg,rgba(91,73,205,.9),rgba(17,97,254,.8) 95.58%);
-    background-color: #0f0e0f;
+    /*background: linear-gradient(264.99deg,rgba(91,73,205,.9),rgba(17,97,254,.8) 95.58%);*/
```
Comment (MathisGD) [inline app/src/components/navbar.module.css]: Remove this

File: app/src/components/navbar.module.css [hunk 5]
```diff
@@ -1,6 +1,7 @@
...
+    background: var(--background);
+    /*background-color: #0f0e0f;*/
```
Comment (MathisGD) [inline app/src/components/navbar.module.css]: Same here

File: app/src/components/typography.css [hunk 6]
```diff
@@ -0,0 +1,66 @@
+.typography--variant-h1 {
```
Comment (MathisGD) [inline app/src/components/typography.css:1]: Why did you delete the `module` ?

**Review by MathisGD — COMMENTED** [review]
Nice job !
I will just change the logo so that he can also change of color.

File: app/src/components/typography.css (hunk 6 above)
Comment (0xnikter) [reply app/src/components/typography.css:1]: Cuz styles work differently when you use .module.css. In this case you have to import "styles" variable and put "styles.className" in each component. We just want to import all CSS in the whole component

Comment (MathisGD) [reply app/src/components/typography.css:1]: Okay :+1:

**Review by MathisGD — COMMENTED** [review]
Perfect for the moment !
I will merge it, so that we can start the connexion with contracts
